interfaces/cython/cantera/examples/surface_chemistry/sofc.py

```diff
@@ -89,21 +89,14 @@ def NewtonSolver(f, xstart, C=0.0):
 # Anode-side phases
 #####################################################################
 
-# import the anode-side bulk phases
-gas_a, anode_bulk, oxide_a = ct.import_phases(
-    'sofc.yaml', ['gas', 'metal', 'oxide_bulk'])
-
-# import the surfaces on the anode side
-anode_surf = ct.Interface('sofc.yaml', 'metal_surface', [gas_a])
-oxide_surf_a = ct.Interface('sofc.yaml', 'oxide_surface', [gas_a, oxide_a])
-
-# import the anode-side triple phase boundary
-tpb_a = ct.Interface('sofc.yaml', 'tpb', [anode_bulk, anode_surf, oxide_surf_a])
+# import the anode-side triple phase boundary and adjacent phases
+tpb_a = ct.Interface("sofc.yaml", "tpb")
+anode_surf, oxide_surf_a, anode_bulk = tpb_a.adjacent.values()
+oxide_a, gas_a = oxide_surf_a.adjacent.values()
```

Comment:
I think getting entries from the dictionary by key is easier to follow. (here and elsewhere)

Reply:
I was kind of on the fence here. Unpacking the `values()` like this was partly motivated by a suggestion you made in Cantera/enhancements#103. Going the more explicit route ends up being not that much less verbose than the old syntax in the cases where you actually want to assign variables for all the adjacent phases, but so it goes.

Reply:
True, I did suggest to unpack from a tuple (which imply a given order). I think your avenue of upgrading this to a dictionary makes a given order less obvious, and allows for better 'self-documentation'. If you want to keep things short, there's always the option of using a generator (although they're harder to read).